src/utils/schema/resolvers/Query/all.ts

```diff
@@ -16,10 +16,29 @@ import { getAccessibleFields } from '@utils/form';
 import buildCalculatedFieldPipeline from '@utils/aggregation/buildCalculatedFieldPipeline';
 import { flatten, get, isArray } from 'lodash';
 import { logger } from '@services/logger.service';
+import config from 'config';
+import i18next from 'i18next';
 
 /** Default number for items to get */
 const DEFAULT_FIRST = 25;
 
+/**
+ * Checks pagination maximum limit of data.
+ * Throw pagination maximum limit cross than error.
+ *
+ * @param maxLimit Question maxLimit number
+ */
+export const paginationMaxLimit = (maxLimit: number): void => {
```

Comment:
@ketanreliefapps 
this comment is just for information, you don't need to work on it
it does not make sense to put this code there
it is a shared method, not a specific code of the query/all
please make sure for next tickets to take that into account when deciding where to put code

Reply:
Sorry, Okay